File: MaWPictures/src/main/java/us/mikeandwan/photos/data/Comment.java
package us.mikeandwan.photos.data;

import java.util.Date;


@SuppressWarnings("ALL")
public class Comment {
    private Date _entryDate;

    public Date getEntryDate() {
        return _entryDate;
    }

    public void setEntryDate(Date entryDate) {
        _entryDate = entryDate;
    }

    private String _commentText;

    public String getCommentText() {
        return _commentText;
    }

    public void setCommentText(String commentText) {
        _commentText = commentText;
    }

    private String _username;

    public String getUsername() {
        return _username;
    }

    public void setUsername(String username) {
        _username = username;
    }
}

File: MaWPhotos/src/main/java/us/mikeandwan/photos/api/Comment.kt
package us.mikeandwan.photos.api

import kotlinx.serialization.Contextual
import java.util.*
import kotlinx.serialization.Serializable

@Serializable
data class Comment(
    @Contextual val entryDate: Date,
    val commentText: String,
    val username: String
)
